python-api/tests/integrationtests2/baseclass.py

```diff
@@ -0,0 +1,118 @@
+"""
+Contains the Base class for integration tests.
+"""
+
+import sys
+import os
+import logging
+
+sys.path.insert(0, os.path.abspath(os.path.join(__file__, "../../../")))
+from gstswitch.server import Server
+from gstswitch.helpers import TestSources  # PreviewSinks
+from gstswitch.controller import Controller
+from gi.repository import GLib
+
+# PATH = os.getenv("HOME") + '/gst/stage/bin/'
+PATH = '../tools/'
```

Comment:
This is fine for now, but we really need a better "find gst-switch" style function.